src/Extensions/Logging/Internal/OmexLoggerProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class OmexLoggerProvider : ILoggerProvider, ISupportExternalScope
+	{
+		public OmexLoggerProvider(ILogEventSender logsEventSource, IExternalScopeProvider defaultExternalScopeProvider)
```

Comment:
It would be used as default if `SetScopeProvider` won't be called to provide different implementation, unfortunately for some reason, IExternalScopeProvider not register in DI by default implementation, so I've added it by preserving the way of overriding in both ways